Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/CompileWithArguments.dfy

```diff
@@ -1,7 +1,6 @@
 // NONUNIFORM: Multiple testing scenarios, highly backend sensitive, testing CLI
 // RUN: %verify "%s" > "%t"
 // RUN: %run --no-verify --target:cs "%s" Csharp 1 >> "%t"
-// RUN: %run --no-verify --target:cpp --unicode-char:false "%s" Cpp Yipee >> "%t"
```

Comment:
We should keep these, otherwise we're reducing testing coverage for the cpp backend.

Reply:
They are kept. I've swapped `comp/CompileWithArguments.dfy` and `unicode/comp/CompileWithArguments.dfy`. I realize this is a confusing change in this PR, and it would have been better to do it before or after, but I think it might be bit of a hassle to extract it as a separate PR now.

Reply:
Ah got it, that's just fine.